spi/src/main/java/org/arquillian/cube/spi/metadata/HasMappedPorts.java

```diff
@@ -0,0 +1,13 @@
+package org.arquillian.cube.spi.metadata;
+
+
+public interface HasMappedPorts extends CubeMetadata {
+    
+    String getIP();
+
+    Integer forTargetPort(int targetPort);
```

Comment:
I think this needs to return some kind of IP/Port Pair. It would allow to communicate with different ports on the same cube via different interfaces, e.g. externally routed or internally forwarded depending on the port. Maybe URI could do? tcp://ip:port? udp://ip:port?